lib/pure/parsecfg.nim

```diff
@@ -306,6 +370,7 @@ proc getSymbol(c: var CfgParser, tok: var Token) =
     add(tok.literal, c.buf[pos])
     inc(pos)
     if not (c.buf[pos] in SymChars): break
+  tok.literal = tok.literal.strip
```

Comment:
I'd rather fix the lexer so that it doesn't producing leading/trailing whitespace. ``.strip`` feels a bit hacky.

Reply:
how about strip it in `parseSymbol`?

Reply:
Now it only strips trailing whitespace.
Leading space is skipped by `skip` function already.

I don't where I should fix. Any hints?

Reply:
the trailing space comes from:
```nim
foo   :text
time = hello
```
the space before special char such as `:`, `=` and so on
And the space before `newLine`